nextjs-logs-manager/app/components/EditLogForm.tsx

```
@@ -0,0 +1,163 @@
+                  defaultDate={new Date(value)}
+                  onSelectedDateChanged={(e) => {
+                    const dateStr = e.toJSON();
+                    const i = dateStr.indexOf("T");
+                    const selectedDateStr = dateStr.slice(0, i);
+                    // console.log(date);
+                    onChange(selectedDateStr);
+                  }}
+                />
+              )}
+            />
+            {errors && errors.date && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)